README.md

```diff
@@ -24,9 +24,7 @@ $ npm i && npm run live
 
 ## What is first-class function?
```

Comment:
Can we add an 'a'?
> What is a first-class function?

Reply:
@astroash Github's review things are so confusing. Not sure if you saw this one too? The title is just missing an 'a', then we're good to go 👍

Reply:
Oops my bad, I missed this review! Now pushed with real grammar 🍎